spec/views/cases/filters/index_html_slim_spec.rb

```diff
@@ -0,0 +1,273 @@
+                                 responding_team: responding_team)
+                            .decorate }
+  let(:accepted_case)   { create(:accepted_case, :flagged_accepted,
+                                 responder: responder)
+                            .decorate }
+  
+  let(:search_query) { build :search_query }
+  let(:unflagged_case) { create(:case,
+                                responding_team: responding_team)
+                             .decorate }
+
+  let(:ovt_foi_trigger_case)    { create(:accepted_ot_ico_foi).decorate }
+
+  let(:branston_responder) { find_or_create :branston_user }
+  let(:offender_sar_case) { create :offender_sar_case}
+  let(:offender_sar_case_third_party_name) { create :offender_sar_case, 
+                                                      third_party_relationship: 'family',
+                                                      third_party: true, 
+                                                      third_party_name: "third_party_name"}
+  let(:offender_sar_case_third_party_company) { create :offender_sar_case, 
+                                                        third_party_relationship: 'solicitor',
+                                                        third_party: true, 
+                                                        third_party_company_name: "third_party_company_name"}
+
+  let(:request)         { instance_double ActionDispatch::Request,
+                                          path: '/cases/open',
+                                          fullpath: '/cases/open',
+                                          query_parameters: {},
+                                          params: {}}
+
+  before do
+    assign(:global_nav_manager, GlobalNavManager.new(responder,
+                                                     request,
+                                                     Settings.global_navigation.pages))
+    allow(request).to receive(:filtered_parameters).and_return({})
+    assign(:homepage_nav_manager, GlobalNavManager.new(responder,
+                                                     request,
+                                                     Settings.homepage_navigation.pages))
+  end
+
+  def login_as(user)
+    allow(view).to receive(:current_user).and_return(user)
+  end
+
+  it 'displays the cases given it' do
+    login_as responder
+    assigned_case
+    accepted_case
+    unflagged_case
+    ovt_foi_trigger_case
+
+    assign(:cases, PaginatingDecorator.new(Case::Base.all.page.order(:number)))
+    assign(:state_selector, StateSelector.new( {} ))
+    assign(:query, search_query)
+    assign(:action_url, '/cases/open')
+    assign(:current_tab_name, 'open')
+
+    disallow_case_policy :can_add_case?
+
+    render
+    cases_page.load(rendered)
+
+    zeroth_case = cases_page.case_list[0]
+    expect(zeroth_case.number.text).to eq "Case number #{ovt_foi_trigger_case.original_case.number}"
```

Comment:
Might be nice to add these parts to named methods just so it's clearer to read, something like:
```ruby
def case_expectations(case, values)
    expect(case.number.text).to eq values[:case_num]
    expect(case.type.text).to eq values[:type_text]
    expect(case.flag.text).to eq values[:flag_text]
    expect(case.request_detail.text) values[:request_text]
    expect(case.draft_deadline.text).to eq values[:request_deadline_text]
   # etc...
end
```

and then just look up each case and create a hash for each - would save on some of the logic repetition 👍